packages/product-components/src/components/SelectAssetModal/NetworkTabs.tsx

```diff
@@ -0,0 +1,138 @@
+import { AssetLogo, Row, useElevation } from '@trezor/components';
+import {
+    borders,
+    Elevation,
+    mapElevationToBackground,
+    mapElevationToBorder,
+    spacings,
+    spacingsPx,
+    typography,
+} from '@trezor/theme';
+import { SelectAssetNetworkProps, SelectAssetSearchCategoryType } from './SelectAssetModal';
+import styled, { DefaultTheme } from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+
+interface CheckableTagProps {
+    $variant: 'primary' | 'tertiary';
```

Comment:
Could you use it this way please? 🙏 

```
export const tagVariants = ['primary', 'tertiary'] as const;
export type TagVariant = Extract<UIVariant, (typeof tagVariants)[number]>;
```

we use similar approach everywhere we use variants

Reply:
Done! Thanks